jsapp/js/components/activity/formActivity.scss

```diff
@@ -0,0 +1,91 @@
+@use 'scss/sizes';
+@use 'scss/mixins';
+@use 'scss/breakpoints';
+@use 'scss/colors';
+
+.main-container {
+  display: flex;
+  flex-direction: column;
+  overflow: hidden;
+  background-color: colors.$kobo-white !important;
+
+  .header {
+    width: 100%;
+    display: flex;
+    gap: sizes.$x8;
+    align-items: center;
+    padding: 0 sizes.$x20;
+
+    h1 {
+      color: colors.$kobo-gray-800;
```

Comment:
It should have `margin: 0` to reset the `normalize.css` value

Reply:
And the color doesn't really change anything, as it's the same as the color for `html`